themes/classic/_dev/js/customer.js

```diff
@@ -0,0 +1,18 @@
+import $ from 'jquery';
+
+function initRmaItemSelector () {
+  $('#order-return-form table thead input[type=checkbox]').on('click', function () {
+    var checked = $(this).prop('checked');
+    $('#order-return-form table tbody input[type=checkbox]').each(function (_, checkbox) {
+      $(checkbox).prop('checked', checked);
+    })
+  })
+}
+
+function setupCustomerScripts () {
+    if ($('body#order-detail')) {
```

Comment:
An empty array is truthy in Javascript and since jQuery always returns arrays even when there are no elements this condition is always true.

You should check `$('body#order-detail').length > 0`.